src/main/resources/Notes/WebHome.xml

```diff
@@ -26,7 +26,7 @@ $services.localization.render('notes.app.info')
 {{/velocity}}
 {{/info}}
 
-{{spaceindex count="5"/}}</content>
+{{documentTree/}}</content>
```

Comment:
I would like a after screenshot. Also on default XE the parameters are 
`{{documentTree showTranslations="false" showAttachments="false" /}}`